client/components/domains/map-domain-step/index.jsx

```diff
@@ -183,9 +191,23 @@ var MapDomainStep = React.createClass( {
 	},
 
 	handleValidationErrorMessage: function( domain, error ) {
-		let message;
+		let message,
+			errorLevel = 'error';
```

Comment:
It'd be great if the new variables' names in this file could be consistent with the ones in the other file.
So, we have `state.errorLevel` and `errorLevel` here, while in the other file it's `state.noticeSeverity` and `severity`. I prefer the second version, but just keeping them consistent is what's important. Maybe these components could be refactored in the future, since they seem to have some common code/patterns and common naming helps spotting them :)